tpl/tplimpl/embedded/templates/shortcodes/__h_simple_assets.html

```diff
@@ -3,19 +3,19 @@
 {{/* Only include once */}}
 {{  .Page.Scratch.Set "__h_simple_css" true }}
 <style>
-.__h_youtube {
+.__h_youtube, .s_youtube_simple {
    position: relative;padding-bottom: 56.23%;height: 0;
    overflow: hidden;
    max-width: 100%;
    background: #000;
    margin: 5px;
 }
-.__h_youtube img {
+.__h_youtube img, .s_youtube_simple img {
```

Comment:
The idea (which may or not be a good one) was to give users a stable class to match with their external stylesheets. It does not make sense to use it in the inline styles. And not in this PR, anyhow.

Reply:
Ok.